### DISCUSSION TITLE
Add fel lotus to badge vendor

### DISCUSSION THREAD

**Discussion by goldritens** [chromiecraft/chromiecraft#7199] [Suggestions]
Heroic queue is dead and buried 6 feet under, and nobody farms herbs. Adding fel lotus to the badge vendor for like 20 badges solves both problems. Another alternative could be to rep lock them with like Cenarion or something and make them 25g.

**Comment by bigdeekpumper** [comment]
I'm not against the idea of adding more items to be purchased with BoJs, but why fel lotus specifically? Doing so sets a precedent that can render all gathering professions useless, because why stop there and not sell khorium ores, eternium ore, hard knothide leather, etc. 

I don't know if people aren't gathering herbs, but supposing they aren't  then it's an economy problem that should regulate itself over time.
(As in: if fel lotus/flasks become expensive it'll incentivize people to farm them. But then again, I'm not tracking this situation, and I'm not against an intervention if the economy is totally effed up because of server pop or any other reason.)

Now if you really want to get flasks with badges I believe it would be more fair if you could purchase either Marks of Illidari (which require effort to turn into Shattrath flasks: exalted rep with Aldor or Scryer) or Shattrath flasks directly (no effort required, but flask is still limited to TBC raids).

**Comment by EllendeUndead** [comment]
Give more gold for a hc dungeon run and help the whole economy, instead of giving them the goods right away (you can buy fel lotus with gold you know). If you'd do that, there would be less gold on the server (farming + selling -> buying other stuff from someone else). Less gold on the server = less happening on the server in general. So give incentive to buy stuff (gold for hc dungeons) = more farming on the server of any type.
